### PR TITLE
all make commands copy files to build/output

### PR DESCRIPTION
**Pull request by MattAtkinson**
Signed-off-by: Matt Atkinson <matt.atkinson@imgtec.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -61,8 +61,8 @@ clean_feeds:
 	cd ../dist/openwrt; \
 	rm -rf .config feeds.conf tmp/ feeds;
 
-clear_output:
+clean_output:
 	rm -rf output/
 
 .PHONY: clean_binaries
-clean_binaries: clear_output copy_openwrt copy_contiki
+clean_binaries: clean_output copy_openwrt copy_contiki
```
Comment (abhijit-mahajani-imgtec) [inline Makefile]: why clean_binaries rule also copying things?

File: Makefile [hunk 2]
```diff
@@ -52,4 +61,8 @@ clean_feeds:
 	cd ../dist/openwrt; \
 	rm -rf .config feeds.conf tmp/ feeds;
 
-clean: clean_openwrt clean_feeds clean_contiki
+clear_output:
+	rm -rf output/
+
+.PHONY: clean_binaries
```
Comment (abhijit-mahajani-imgtec) [inline Makefile]: clean_binaries should really be called from clean rule

**Comment by abhijit-mahajani-imgtec** [comment]
Approved

**Comment by abhijit-mahajani-imgtec** [comment]
Verified

**Comment by nikhil-zinjurde-imgtec** [comment]
QA Verified

**Pull request merged**